crates/loam-cli/src/commands/mod.rs

```diff
@@ -39,6 +40,7 @@ impl Root {
         match &mut self.cmd {
```

Comment:
Ah so since this a mutable reference you can change `dev_info.run` to use `(&mut self)`, which removes the need to clone in the `build` method.